funnel/views/membership.py

```diff
@@ -139,7 +139,7 @@ def new_member(self):
                 return (
                     {
                         'status': 'error',
-                        'message': _("The new member could not be added"),
+                        'error_description': _("The new member could not be added"),
```

Comment:
This is unhelpful as it doesn't state why. However, it's a pre-existing concern.